index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Landing Page</title>
+</head>
+
+<body>
+    <header id="header">
+        <nav id="nav-bar">
+            <ul id="navigation-bar"></ul>
+            <div>
+                <img id="header-img" src="gnysw2objzekbagrqiax.png" alt="Microverse">
+            </div>
+            <form action="index.html">
+			<ul>
+            <li><button type="button" class="nav-link">Home</button></li>
+			</ul>
+			</form>
+			<form action="contact.html">
+			<ul>
+            <li><button type="button" class="nav-link">Contact Us</button></li>
+			</ul>
+			</form>
+			<form action="review.html">
+			<ul>
+            <li><button type="button" class="nav-link">Reviews</button></li>
+            </ul>
+			</form>
+        </nav>
+    </header>
+    <div class="flex-container">
+        <div>1</div>
+        <div>2</div>
+        <div>3</div>
+        <div>4</div>
+        <div>5</div>
+    </div>
```

Comment:
It's great that you included flex-container here? Can you kindly make sure that instead of <div> class, you use <section> class, which would be more inline with semantic style? Additionally, can you also define class or id attribute to the individual elements (1,2,3,4), so that you can assign style properties in css file?